Orange/widgets/visualize/utils/plotutils.py

```diff
@@ -0,0 +1,151 @@
+from AnyQt.QtWidgets import QScrollArea, QSizePolicy, QAction
+from AnyQt.QtCore import Qt
+from AnyQt.QtGui import QKeySequence
+
+from Orange.widgets import gui
+
+
+JITTER_SIZES = [0, 0.1, 0.5, 1, 2, 3, 4, 5, 7, 10]
+
+
+def create_scroll_area(master):
```

Comment:
This does not belong to plotutils. Feels like `gui.py`, though I'm not sure it's general enough. But if it is, it's not related to plots only.